### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by Khaleelur-Rahman**
Fixed nits in DG and added Activity diagram for `add_tag` feature.

Updated PPP and AboutUs.md.

### CODE REVIEW

File: docs/team/khaleelur-rahman.md [hunk 1]
```diff
@@ -83,4 +85,4 @@ Given below are my contributions to the project:
 <br>
 
 * **Contributions beyond the project team**:
-  - Reviewed a different group's project during PE-D: [PE-D Bugs](https://github.com/Khaleelur-Rahman/ped/issues)
+  - Reviewed a different group's project during PE-D: [PE-D Bugs](https://github.com/Khaleelur-Rahman/ped/issues). Some of the
```
Comment (rionshocker) [inline docs/team/khaleelur-rahman.md:88]: Is the `Some of the` meant to be here?

**Review by rionshocker — CHANGES_REQUESTED** [review]
LGTM! Just a minor nit.

File: docs/team/khaleelur-rahman.md (hunk 1 above)
Comment (Khaleelur-Rahman) [reply docs/team/khaleelur-rahman.md:88]: Oops, good catch. Thank you for spotting!

**Review by rionshocker — APPROVED** [review]
LGTM!

**Pull request merged**